packages/terra-form-textarea/src/Textarea.module.scss

```diff
@@ -1,10 +1,10 @@
 :local {
   .textarea {
     background-color: var(--terra-form-textarea-background-color, #fff);
-    background-image: var(--terra-form-textarea-focus-animation-background, none);
+    background-image: var(--terra-form-textarea-focus-animation-background-image, none);
```

Comment:
nit: I know we are using background-image for focus animation, and this variable name helps clarify our use of it, though I think we can drop the focus-animation part from this variable name mainly to be consistent with our other variable names.

Reply:
https://github.com/cerner/terra-core/pull/2125/commits/8a9967b6e0cf7ff56d7f2d657946e5c82e1c59fc#diff-ee126436faa9db1c57ae86d278387a97R4